lib/rspec/que/queue_up.rb

```diff
@@ -1,89 +1,155 @@
 require 'rspec/mocks/argument_list_matcher'
+require 'time'
 
 module RSpec
   module Que
     module Matchers
       class QueueUp
         def initialize(job_class = nil)
+          @matchers = [QueuedSomething.new]
+          @matchers << QueuedClass.new(job_class) if job_class
           @job_class = job_class
+
+          @stages = nil
```

Comment:
I'm moving @stages=[] to here instead of in #matches?, which will keep all the instance variables initialized once and will maintain the property that they are all listed in `#initialize` (which is nice for figuring out what goes on in a gem)